DQM/TrackingMonitor/python/trackingRecoMaterialAnalyzer_cfi.py

```diff
@@ -18,6 +18,9 @@
                                         PropagatorOpposite = cms.string("RungeKuttaTrackerPropagatorOpposite")
 )
 
+from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
+trackingPhase2PU140.toModify(materialDumperAnalyzer, TrackerRecHitBuilder='WithTrackAngle')
```

Comment:
I'd suggest to use `phase2_tracker` sub-era instead. All other TTRHBuilder instances are customized with that, and not using templates is more a property of the "detector", than a property of "tracking", even if we have only 1 phase2 tracking configuration at the moment.